src/Services/Device.vala

```diff
@@ -286,6 +286,9 @@ public class Power.Services.Device : Object {
             return _("Calculating…");
         }
 
+        if (percent == 100 && is_charging) {
+            return _("Fully charged");
+        }
```

Comment:
```suggestion
        }

```

I personally want a newline here for readable code, because the variable `info` in the next line is irrelevant with this if statement